source/_components/device_tracker.google_plus.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Google Plus"
+description: "Instructions how to use Google Plus to track Android devices in Home Assistant."
```

Comment:
I think the Google plus location is updated from any device with the G+ app and not only Android devices.